.github/workflows/_build-aleph-e2e-client-image.yml

```diff
@@ -16,11 +21,13 @@ on:
 jobs:
   check-vars-and-secrets:
     name: Check vars and secrets
+    if: ${{ inputs.push == true }}
```

Comment:
Please put `true` in single quotes.  In if conditions, it's string as well.  Been there, had problems.

Reply:
In this workflow, there's `type: boolean`, should I change it to `type: string` ?

Reply:
I know, but you'd still need to compare it to string.  I have checked other workflows but please double check github-actions repo again.